src/App.jsx

```diff
@@ -1,4 +1,4 @@
-import { Routes, Route } from 'react-router-dom';
+import {Routes, Route} from 'react-router-dom';
```

Comment:
Is your prettier broken? Why is there no empty space between { } ?

Reply:
update Prettier settings